### ISSUE TITLE
Unsupported operation _Switch

### ISSUE THREAD

**Issue by JHMGo** [NVIDIA/TensorRT#485]
I tried to use my program in c++ to reach the same function as uff_resnet50.py in 

> TensorRT-6.0.1.5/samples/python/introductory_parser_samples
and use my own dataset for training. This uff model could not be loaded and reported:
```
[E] [TRT] UffParser: Validator error: bn5c_branch2a/cond/Switch: Unsupported operation _Switch
[E] [TRT] Network must have at least one output
```
However，before that, I have already test resnet50-infer-5.uff in 

> TensorRT-6.0.1.5/data/resnet50
with my own program and it could load model and return labels and probability.
I don't know why my model cannot pass, just because I train my model in tensorflow+keras rather than only tensorflow?

Environment:
TensorRT 6.0.1.5
CUDA:10.0
CUDNN:7.6.5
NVIDIA driver:440.31
Device:NVIDIA P4000
Tools for training model:Tensorflow 1.14.0+keras 2.1.3

**Comment by SergeLiu** [comment]
@JHMGo when I converting the model from the keras .h5 model to .pb then to .uff, I  need to add a custom wrapper but when building engine from .uff, custom layer is not supported, finally I delete all my custom layer from keras model

**Comment by kevinch-nv** [comment]
UFF has been deprecated, so closing this issue. We recommend going through the tf2onnx path if you plan on running future tensorflow models in TensorRT.
